### ISSUE TITLE
Not working in Android in proper way

### ISSUE THREAD

**Issue by ssharmass** [planetoftheweb/angularMobile#1]
Hi ,
  You code is only working on iOS OS and web browser but in Android it is not working in following area :
1. Hide search everywhere
2. Tab is displaying at top
3. White screen at top and sometime above the list 
4. On click over list , it is displaying calender not artist list or back 
5. all above for android only.

**Comment by planetoftheweb** [comment]
Can you provide screenshots so I can see the issue. My suggestion is to try removing the CSS file and see if it works. It should be ugly but maybe work better.

**Comment by ssharmass** [comment]
Hi,
This is pic of what i got in mobile, even I tested on ripple bit , in which when i select iPhone series Phone , it works but select android it doesn't.

![screenshot_2015-07-03-15-15-07](https://cloud.githubusercontent.com/assets/13134188/8497017/6a059b82-2197-11e5-97a0-1d81453ade22.png)

I got another issue , not regarding to your theme , ng-click issue , when I  mentioned somewhere in function alert it is triggered that two times and block navigation of page . Have you any idea why it is not doing this.

Thank & Reagrds
Shivam

**Comment by ssharmass** [comment]
![screenshot_2015-07-03-15-15-12](https://cloud.githubusercontent.com/assets/13134188/8497133/513c868c-2198-11e5-9639-8ee299e99b8a.png)
![screenshot_2015-07-03-15-30-56](https://cloud.githubusercontent.com/assets/13134188/8497169/9e5339e8-2198-11e5-9c94-c6377ebf34c5.png)

**Comment by ssharmass** [comment]
I followed your css idea but Its so ugly UI, Actually I was making app using ionic then I found your theme to refer , but they got something that I mentioned above.I really want to sort out this issues , that is why generate issue, I hope you will have solution.

Thanks

**Comment by ssharmass** [comment]
Hi,
  I have found solution and it is working. I have another issue that Pull to Refresh not working.
  Let me know this time.

Regards
Shivam

**Comment by planetoftheweb** [comment]
Yeah, I submitted a ticket to ionic. I think this is something wrong with the current version of their API. It’s obviously working in the version that I used in the course, but an update broke it. They haven’t replied.

-- 
Ray Villalobos

On September 15, 2015 at 2:52:54 AM, ssharmass (notifications@github.com) wrote:

Hi,
I have found solution and it is working. I have another issue that Pull to Refresh not working.
Let me know this time.

Regards
Shivam

**Comment by AlicanGultekin** [comment]
Hi Ray, I've just finished the course on Lynda and I was wondering about the same issue regarding Android. Have you heard back from Ionic yet?

**Comment by drothv** [comment]
Hi, I have a problem at this point of the exercise, I used your code for index.html, data.json and some jpg images, but when it gets to ion-item, it does nothing. Do you think it is something to do with versions?

 <ion-item 
          ng-repeat='item in artists | filter: query'
          class="item-thumbnail-left 
            item-text-wrap">
            <img src="img/{{item.shortname}}_tn.jpg"
              alt="{{ item.name }} Photo">
            <h2>{{item.name}}</h2>
            <h3>{{item.reknown}}</h3>
            <p>{{item.bio | limitTo: 100}}
            {{ item.bio.length > 150 ? '&hellip;' : '' }}</p>
          </ion-item>

**Comment by drothv** [comment]
Hi, I have a problem at this point of the exercise, I used your code for index.html, data.json and some jpg images, but when it gets to ion-item, it does nothing. Do you think it is something to do with versions?

**Comment by rashid-mycnw** [comment]
How can I change my ionic (and other dependencies) version to the one you used?
